CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## [Unreleased]
+- Buy ERC20 tokens using Moonpay
+- Remove CryptoCompare; use Coingecko for latest conversion rates. Fixes rate limiting issues, especially for VPN/Tor users.
+- Bundle BitBox02 v9.5.0 firmware
+- Run BitBoxApp installer as admin by default on Windows
+- Close a running BitBoxApp instance on Windows when installing an update to ensure success
+
```

Comment:
Consider adding another item: More descriptive error when TOR connection fails.

Reply:
Saw too late: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1219

You can also "request changes" so I can't merge :)